topsort-api-v2.yml

```diff
@@ -355,10 +359,13 @@ components:
             - missing_slots
             - no_products
             - no_purchase_items
+            - purchase_item_quantity_less_or_equal_than_zero
+            - resolved_bid_id_not_found
```

Comment:
Do we have a custom error code for timeout related errors?

Reply:
Actually now I'm not sure. We might have a mix of both, but anyways, there's no timeout-specific error, I assume we return an internal_server_error in that case? @sk-